webui/react/src/routes/utils.ts

```diff
@@ -181,6 +181,14 @@ export const linkPath = (aPath: string, external = false): string => {
   return path;
 };
 
+const routeById = (id: string): RouteConfig => {
+  // OPTIMIZE: We might want to change the route data structrue, or precompute
+  // to speed up this process.
```

Comment:
**suggestion:** Probably best to pre-compute instead of data structure change since the order of `routes` does matter in terms of path resolution. But a simple record<id, path (or full route config)> map generated once from `routes` should do the trick.

Reply:
I won't be doing this in this PR. should I take out the comment? it's nothing critical in terms of performance